doc/REST-interface.md

```diff
@@ -31,6 +31,11 @@ With the /notxdetails/ option JSON response will only contain the transaction ha
 
 Given a block hash: returns <COUNT> amount of blockheaders in upward direction.
 
+#### Blockhash
+`GET /rest/blockhash/<HEIGHT>.<bin|hex|json>`
+
+Given a height: returns blockhash of block in main chain at height <HEIGHT>.
```

Comment:
`<HEIGHT>` doesn't appear in the Markdown preview.

Reply:
Fixed